### PR TITLE
Error creating event in full form

### PR DESCRIPTION
**Pull request by nader77**
#341

### CODE REVIEW

File: capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/main.js [hunk 1]
```diff
@@ -238,6 +238,15 @@ angular.module('c4mApp')
       }
     };
 
+    /**
+     * @TODO: Please document this.
```
Comment (nader77) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/main.js]: @HelenaEksler ^^

**Comment by HelenaEksler** [comment]
@nader77 Added docs.

**Comment by nader77** [comment]
Update: 
* Fixed creating event in full form (Resource was not returning the entity details).
* Fixed errors colors
* Added scroll to element errored when upon submitting a quick post (Wasn't clear that there was an error before).
* Removed unnecessary validations on creating a full form.
* Added ``ng-cloak`` to the quick post form again. 

Workflow with error scroll and event full form:
errors

**Comment by nader77** [comment]
Errors because of body ID, Will fix.

**Comment by nader77** [comment]
@amitaibu This PR is ready for code review.

File: capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module [hunk 2]
```diff
@@ -200,6 +200,7 @@ function c4m_restful_quick_post_preprocess_html(&$vars) {
   }
   $vars['attributes_array']['ng-app'][] = 'c4mApp';
   $vars['attributes_array']['ng-controller'][] = 'MainCtrl';
+  $vars['attributes_array']['ng-cloak'][] = 'true';
```
Comment (amitaibu) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: can we move this to not appear on the ``<body>`` but only on the main content?

Comment (nader77) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: @amitaibu I can move this to the ``page.tpl`` and write it as normal HTML ?

Comment (amitaibu) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: or maybe just move this to ``preprocess_page``

Comment (nader77) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: It's not working in ``preprocess_page``, Should i add it to the tpl or keep it in ``preprocess_html`` ?

Comment (amitaibu) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: keep as is.

**Pull request merged**